lib/SearchAndSort/SearchAndSort.js

```diff
@@ -392,22 +406,25 @@ class SearchAndSort extends React.Component {
     const message = !(resource && resource.hasLoaded) ? 'Loading...' :
           `No results found${maybeTerm}. Please check your ${maybeSpelling}filters.`;
     const sortOrder = this.queryParam('sort') || '';
-    const filters = filterState(this.queryParam('filters'));
 
     return (
       <Paneset>
         <SRStatus ref={(ref) => { this.SRStatus = ref; }} />
         {/* Filter Pane */}
-        <Pane id="pane-filter" dismissible defaultWidth="16%" paneTitle="Search & Filter" header={searchHeader}>
-          <FilterGroups
-            config={filterConfig}
-            filters={filters}
-            onChangeFilter={this.onChangeFilter}
-            onClearFilter={this.onClearFilter}
-            onClearAllFilters={this.onClearAllFilters}
-            disableNames={disableFilters}
-          />
-        </Pane>
+        { filterPaneIsVisible ?
```

Comment:
This requires completely re-rendering the filters when toggling it to visible. Would it be preferable to `display:none` or even just layer it underneath the results pane?

Reply:
Thanks and I see your point. But given that this isn't something that redraws often, and will never be part of a change-state-redraw-change-state-redraw loop, I don't think it's a big issue. Profiling it (even in webpack dev server) didn't seem to show any red flags.

My argument for writing it this way is that it's understand and it's more idiomatically Reactish.